### ISSUE TITLE
(manifest:start) does not return on Win7 64-bit SBCL

### ISSUE THREAD

**Issue by ghost** [gigamonkey/manifest#9]
Hi Peter

I quickloaded "manifest" and wanted to try it out. If I call (manifest:start) nothing happens, REPL seems to hang, no string with URL is returned.

My system information:
SBCL 1.0.54
Windows 7 64-bit

Thanks for looking into it.

Matus

**Comment by daimrod** [comment]
IIRC the official windows version doesn't have threads support, you have to use [this fork](https://github.com/akovalenko/sbcl-win32-threads/wiki).

**Comment by ghost** [comment]
Thanks for the link. That made it.

Now I end at a blank page, when I open the URL in my browser. I should see a list of all the systems I installed via quicklisp, shouldn't I?

Matus

**Comment by daimrod** [comment]
No, you should see all systems or packages currently available. A system become available when you load it with ASDF or quicklisp (quicklisp uses ASDF) but also when you define new packages in the REPL or in a file and load it.

**Comment by ghost** [comment]
Ok, but it still does not work. I think it is an issue with this image of SBCL, because I tried with CCL and it works well.

With SBCL the page loads for a long time and then ends with a blank screen.

If it is indeed a SBCL issue, then i am not sure if one shall continue the discussion here...

**Comment by daimrod** [comment]
Hmm I don't know, I think it would be good if you could figure out where manifest hangs out.

**Comment by gigamonkey** [comment]
Is your SBCL built with threading support? (I suspect it may not be by
default on Windows.)

-Peter

On Thu, Jan 5, 2012 at 1:05 AM, mentus
reply@reply.github.com
wrote:

> Hi Peter
> 
> I quickloaded "manifest" and wanted to try it out. If I call (manifest:start) nothing happens, REPL seems to hang, no string with URL is returned.
> 
> My system information:
> SBCL 1.0.54
> Windows 7 64-bit
> 
> Thanks for looking into it.
> 
> Matus
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/gigamonkey/manifest/issues/9

## 

Peter Seibel
http://www.gigamonkeys.com/

**Comment by ghost** [comment]
No, it wasn't. I downloaded the binaries of a forked SBCL version, which supports threads (as suggested by daimrod). I am getting the URL string now, the browser shows only a blank page however. In CCL manifest works as expected.

If I find time I will try to debug it, but may take me time to sort out the problem as I am quite new to Lisp...

**Comment by gigamonkey** [comment]
Hmmm. Let me know if you have any luck. I don't have any Windows
machines to try it on myself.

-Peter

On Thu, Jan 5, 2012 at 5:39 AM, Matus Kmit
reply@reply.github.com
wrote:

> No, it wasn't. I downloaded the binaries of a forked SBCL version, which supports threads (as suggested by daimrod). I am getting the URL string now, the browser shows only a blank page however. In CCL manifest works as expected.
> 
> If I find time I will try to debug it, but may take me time to sort out the problem as I am quite new to Lisp...
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/gigamonkey/manifest/issues/9#issuecomment-3369834

## 

Peter Seibel
http://www.gigamonkeys.com/
